packages/smart/smart.go

```diff
@@ -642,23 +658,24 @@ func (sc *SmartContract) CallContract(flags int) (result string, err error) {
 			node := syspar.GetNode(sc.TxSmart.KeyID)
 			if node == nil {
 				logger.WithFields(log.Fields{"user_id": sc.TxSmart.KeyID, "type": consts.NotFound}).Error("unknown node id")
-				return ``, fmt.Errorf("unknown node id")
+				return retError("unknown node id")
```

Comment:
Errors must be constants, and be declared in the beginning of the file, like var UnknownNodeIDError=errors.New("Unknown node id")